### PR TITLE
BadgeFix

### PR DESCRIPTION
**Pull request by tachorzy**
Badge shop FIXED. Code for credit cards, inventories, and badges refactored.

### CODE REVIEW

File: src/main/java/org/example/BadgeShop.java [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+        createBadgeShopEmbed(badgeList);
+        event.getChannel().sendMessageEmbeds(badgeShopEmbed.build()).queue();
+        badgeShopEmbed.clear();
+    }
+
 }
+
+
+
+//                case "badgeshop":
```
Comment (jnguyen1111) [inline src/main/java/org/example/BadgeShop.java]: left comment in code

File: src/main/java/org/example/Commands.java [hunk 2]
```diff
@@ -180,184 +148,81 @@ public void onMessageReceived(MessageReceivedEvent event){
...
+                    creditCardObject.unequipBadge(event, badgeList, args[1], server);
+                    break;
+                case "clearbadges":
+                    creditCardObject.clearBadges(event, server);
+                    break;
+                case "addbadge":
+                    if(addBadgeObject.addNewBadge(event,server, args[1], args[2], args[3], args[4], Arrays.copyOfRange(args, 5, args.length)));
+                    break;
+                case "inventory":
+                    inventoryObject.printInventoryEmbed(event,server,event.getMember().getId());
```
Comment (jnguyen1111) [inline src/main/java/org/example/Commands.java:222]: Missing a break case here causes inventory to wipe after inventory command is called.

**Review by jnguyen1111 — COMMENTED** [review]
Please make changes to code and update request.

**Review by tachorzy — COMMENTED** [review]
Removed unnecessary comments and fixed missing break statement.

**Review by jnguyen1111 — APPROVED** [review]
Looks good to me.

**Pull request merged**